data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2460 @@
+			"frame rate" 10
+			"random start frame"
+		"drag" .1
+		"acceleration" 1
+		"lifetime" 30
+		"submunition" "asteroid missile"
+			"facing" 30
+		"submunition" "asteroid missile"
+			"facing" -30
+		"submunition" "magic deployer3"
+
+outfit "magic deployer3"
+	weapon
+		sprite "projectile/asteroid twin projectile"
+			"frame rate" 10
+			"random start frame"
+		"drag" .1
+		"acceleration" 1
+		"lifetime" 30
+		"submunition" "asteroid missile"
+			"facing" 30
+		"submunition" "asteroid missile"
+			"facing" -30
+		"submunition" "magic deployer4"
+
+outfit "magic deployer4"
+	weapon
+		sprite "projectile/asteroid twin projectile"
+			"frame rate" 10
+			"random start frame"
+		"drag" .1
+		"acceleration" 1
+		"lifetime" 30
+		"submunition" "asteroid missile"
+			"facing" 30
+		"submunition" "asteroid missile"
+			"facing" -30
+		"submunition" "magic deployer5"
+
+outfit "magic deployer5"
+	weapon
+		sprite "projectile/asteroid twin projectile"
+			"frame rate" 10
+			"random start frame"
+		"drag" .1
+		"acceleration" 1
+		"lifetime" 30
+		"submunition" "asteroid missile"
+			"facing" 30
+		"submunition" "asteroid missile"
+			"facing" -30
+		"submunition" "magic deployer6"
+
+outfit "magic deployer6"
+	weapon
+		sprite "projectile/asteroid twin projectile"
+			"frame rate" 10
+			"random start frame"
+		"drag" .1
+		"acceleration" 1
+		"lifetime" 30
+		"submunition" "asteroid missile" 15
+			"facing" 30
+		"submunition" "asteroid missile" 15
+			"facing" -30
+
+outfit "Asteroid Jump Drive"
+	category "Systems"
+	cost 29631456
+	thumbnail "outfit/asteroid weapon 7"
+	"mass" 20
+	"outfit space" -20
+	"jump speed" 0.2
+	"jump fuel" 200
+	"jump drive" 1
+	"asteroid mount jd" -1
+	"jump effect" "jump drive circle"
+	"jump effect" "jump drive spark 1"
+	"jump effect" "jump drive spark 2"
+	"jump effect" "jump drive spark 3"
+	"jump effect" "jump drive spark 4"
+	"jump effect" "jump drive spark 5"
+	"jump effect" "jump drive spark 6"
+	"jump effect" "jump drive spark 7"
+	"jump effect" "jump drive spark 8"
+	"jump effect" "jump drive spark 9"
+	"jump effect" "jump drive 1"
+	"jump effect" "jump drive 2"
+	"jump effect" "jump drive 3"
+	"jump effect" "jump drive 4"
+	"jump effect" "jump drive 5"
+	"jump effect" "jump drive 6"
+	"jump effect" "jump drive 7"
+	"jump effect" "jump drive 8"
+	"jump effect" "jump drive 9"
+	"jump effect" "jump drive 10"
+	"jump effect" "jump drive 11"
+	"jump effect" "jump drive 12"
+	"jump effect" "jump drive 13"
+	"jump effect" "jump drive 14"
+	"jump effect" "jump drive 15"
+	description "Whatever the mechanism that hides beneath the translucent orb may be, it functions similarly to a jump drive, bridging different parts of the universe together for the cost of merely 200 units of fuel."
+
+# The "ajump drive" effect doesn't exist, but serve to reduce the number of effects spawned: as it always shows the same overall number, by showing empty ones it effectively reduces the visual bloat during the jump
```

Comment:
I'm thinking we can change the jump effect code to allow this to be done in a cleaner manner.

Reply:
Mhm, this was just to adhere to the current limitations